### ISSUE TITLE
Small layout issue if not all parts are published

### ISSUE THREAD

**Issue by stevejburge** [publishpress/publishpress-series#457]
Frontend:
<img width="1136" alt="Screen Shot 2022-03-11 at 10 03 21 AM" src="https://user-images.githubusercontent.com/3868028/157893251-b041e2de-71ec-4612-b762-62799bc03eef.png">

This happens when some posts are published, but some are scheduled:

<img width="280" alt="Screen Shot 2022-03-11 at 10 05 58 AM" src="https://user-images.githubusercontent.com/3868028/157893478-cb84951f-3f9d-47a8-baae-26be49bd8cb8.png">

**Comment by stevejburge** [comment]
@olatechpro Sorry for the confusion.

If some parts are marked "This part is coming soon", then the list breaks:

<img width="449" alt="Screen Shot 2022-03-14 at 10 53 07 AM" src="https://user-images.githubusercontent.com/3868028/158198182-e9e8c681-2050-42e1-aef5-e9ba7f32b354.png">

**Comment by olatechpro** [comment]
@stevejburge 

1.) Are you sure the string "This post is coming soon...." is not hardcoded in your string template as searching the strings in the plugin code doesn't provide any result. So, i believe this string could probably exist in your template settings but you forgot to remove it?

2.) When i scheduled all posts in series and publish one, this is my result:

![scheduled](https://user-images.githubusercontent.com/46832636/158317280-88b05b15-5d06-4987-a313-b38862733d44.jpg)

**Comment by stevejburge** [comment]
@olatechpro This is the updated view on a new site. I think we should leave this for 2.9.0
<img width="945" alt="Screen Shot 2022-03-15 at 12 02 36 PM" src="https://user-images.githubusercontent.com/3868028/158421341-f07b8d76-1514-4abd-be62-511c55aed318.png">

**Comment by olatechpro** [comment]
I think the best approach maybe a new setting

"Unpublished Parts"
- Hide
- Show
If show, enter custom text:

**Comment by olatechpro** [comment]
@stevejburge can you test this feature and update with updated issue?

While testing now, scheduled series are now showing in frontend since they're not published neither is any other status or seeing any coming soon text.

So, can you retest because no reason unpublished series should be showing in the first instance

**Comment by stevejburge** [comment]
@olatechpro Yes, In my testing, it's ignored right now. This may actually be an OK solution for now and we can come back to solve this later.

In this example, Part 9 is scheduled:
<img width="939" alt="Screen Shot 2022-09-23 at 8 27 01 AM" src="https://user-images.githubusercontent.com/3868028/191960100-7ed22f3e-b496-445f-98da-3214e6304a72.png">

On the frontend, Part 9 just doesn't appear:

<img width="295" alt="Screen Shot 2022-09-23 at 8 27 18 AM" src="https://user-images.githubusercontent.com/3868028/191960143-87339182-1ca0-49cc-a633-f032f2ade3f3.png">
